README.md

```diff
@@ -1 +1,101 @@
-Hi
+# Kayaks Next
+
+This webpage serves to promote, show and market the Kayaks produced in La Isla.
+
+## Built With 
+
+- HTML/CSS
+- NextJS
+- Node.JS
+
+
+## Getting Started
+
+If you want a copy of this file go to the github repository and download it from there:
+- [`https://github.com/Metsanpeitto/kayaks_next`](https://github.com/Metsanpeitto/kayaks_next)
+
+Project Kanban Board:
+- [`https://github.com/Metsanpeitto/kayaks_next`](https://github.com/Metsanpeitto/kayaks_next)
+
+
+
+### Live
+
+
+### Setup
+
+```cmd
+git clone  https://github.com/Metsanpeitto/kayaks_next.git
+cd ./kayaks_next
+```
+
+### Install
+
+```cmd
+npm install
+```
+
+### Usage
+
+```cmd
+npm start
+```
+
+### Verify linters
+
+- Autofix linters errors
+
+```cmd
+npx eslint . --fix
+npx stylelint "**/*.{css,scss}" --fix
+```
+
+## To run jest test
+```cmd
+npm run test
+npm run jest
+```
+
+If you are non a technical person you can download the zip file.
+
+- To do that you have to go to the green button that says 'Code' and then press on it.
+- Choose the option 'Download Zip' and wait until it download.
+- Then you need to decompress it.
+- You will need to have Zip compress file software installed in your computer. If you don't have it you can download it from here
+  [`https://www.7-zip.org/download.html`](https://www.7-zip.org/download.html)
+- Do double click on the Capstone-1.zip and choose a folder where to place all teh decompressed files.
+- Once you are done decompressing, you can open the created folder.
+- In it you can double click on the index.html file and choose a browser to open it (For example google Chrome Browser).
+- Now you should be able to see the project.
+- Remember to use the mouse to navigate through the website. Also you can use the keys 'arrow up' and 'arrow down' of your keyboard
+  to scroll up and down. 
+
+
+### Prerequisites
+
+- Node.JS
+
+
+## Authors
+
+👤 **Abraham Rodriguez**
+
+- GitHub: [@metsanpeitto](https://github.com/Metsanpeitto)
+- LinkedIn: [Abraham Rodriguez](https://www.linkedin.com/in/abraham-rodriguez-3283a319a/)
+- Portfolio: [Waldenberg](https://portfolio.waldenberginc.com)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+[![License: CC BY-NC 4.0](https://licensebuttons.net/l/by-nc/4.0/80x15.png)](https://creativecommons.org/licenses/by-nc/4.0/)
+[![License: CC0-1.0](https://licensebuttons.net/l/zero/1.0/80x15.png)](http://creativecommons.org/publicdomain/zero/1.0/)
```

Comment:
Why two?

Reply:
No real reason. This is 3 years old code that I use as a template.
Removed.